### PR TITLE
Os update

### PR DESCRIPTION
**Pull request by marliza3sc**
- Update the Swift version to 5.9 and the target platform to v16.0
- Deleted `View+TextStyle.swift` as we are supporting v16.0 and above

### CODE REVIEW

File: Package.swift [hunk 1]
```diff
@@ -1,13 +1,13 @@
...
 
 import PackageDescription
 
 let package = Package(
     name: "CubeFoundation",
     platforms: [
-        .iOS(.v15),
-        .macOS(.v10_15)
+        .iOS(.v16),
+        .macOS(.v11)
```
Comment (3scsamdavis) [inline Package.swift:10]: Out of interest, do we have to specify mac? 🤔

**Comment by BenShutt** [comment]
@marliza3sc in Github you can add full teams as reviewers 🙌 .

**Review by 3scsamdavis — APPROVED** [review]

File: Package.swift (hunk 1 above)
Comment (BenShutt) [reply Package.swift:10]: Don't have too, it's good to support as much as possible but admittedly this framework uses UIKit in many places so many it's easier to just not support mac

**Comment by BenShutt** [comment]
Any other iOS 16 bits that come to mind @3SCSamDavis ? 👀

**Review by BenShutt — APPROVED** [review]

**Pull request merged**